ui/src/app/components/access-renewal-notification.tsx

```diff
@@ -94,14 +72,15 @@ const AccessRenewalNotification = ({daysRemaining}: {daysRemaining: number}) =>
     }
     </div>
     {/* TODO RW-6618 navigate to Access Renewal pages */}
-    <Button type='primary' style={styles.button} onClick={() => navigateByUrl('access-renewal')}>
+    <Button type='primary' style={styles.button} onClick={() => navigate(['access-renewal'])}>
       <div style={styles.buttonText}>Get Started</div>
     </Button>
   </FlexRow>;
 };
 
-export const AccessRenewalNotificationMaybe = (props: {profile: Profile}) => {
-  const daysRemaining = maybeDaysRemaining(props.profile);
+export const AccessRenewalNotificationMaybe = () => {
```

Comment:
Allows this component to update with the correct number of days as the profile updates

Reply:
This is an interesting one. I updated this before fixing the issue in `navbar`. The way `navbar` was originally getting the profile means that it will only receive one copy of the profile, and never get updated. It was in turn, sending that profile value to the notification for aar.

I updated this code to use a `useStore`. This is a custom (written for this codebase) react hook that will trigger a render whenever a value in the store changes. However, with the fix to navbar we could pass the `profileState` in to this component and it would get updated correctly and trigger renders.

I am going to opt to leave this code as-is. The rationale being: I want to be explicit that this component is accessing data from the `profileStore`. This may be important in the near future as we migrate off of Angular.